integrationTests/vm/esdt/esdtProcess_test.go

```diff
@@ -761,14 +766,13 @@ func TestCallbackPaymentEgld(t *testing.T) {
 
 	nonce, round = integrationTests.WaitOperationToBeDone(t, nodes, 4, nonce, round, idxProposers)
 	_, err = nodes[0].AccntState.GetExistingAccount(firstScAddress)
-	assert.Nil(t, err)
+	require.Nil(t, err)
 
+	txData := testVm.NewTxDataBuilder()
 	//// call first sc with esdt, and first sc automatically calls second sc
```

Comment:
too many // (valid for the rest of the file)

Reply:
Fixed all occurrences in the file.